R-package/DESCRIPTION

```diff
@@ -65,4 +65,5 @@ Imports:
     data.table (>= 1.9.6),
     jsonlite (>= 1.0),
 RoxygenNote: 7.2.2
+Encoding: UTF-8
```

Comment:
This resolves the following warnings from `{roxygen2}`.

> Warning message:
roxygen2 requires Encoding: "UTF-8"
ℹ Current encoding is NA